src/utils/package-manager.ts

```diff
@@ -6,10 +6,15 @@ import * as Sentry from '@sentry/node';
 import { traceStep } from '../telemetry';
 import { getPackageDotJson, updatePackageDotJson } from './clack-utils';
 
-export interface PackageManager {
+// some package managers like Bun support multiple lock files (bun.lockb, bun.lock)
```

Comment:
I know this wasn't defined in the issue we're resolving with this PR but can I ask you to go with a wild approach instead? 😂

We are not using `lockFile` anywhere so having it (and the generics and stuff) is overengineering. I would just remove the lockfile property and inline everything. Does that make sense?

Reply:
I thought so as well, 

I was very anxious to going through all the `lockFile` references just find out it's not used anywhere